### PR TITLE
Fixed translation key and added missing translations

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       |  N/A
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ibexa/admin-ui/blob/main/LICENSE)

The message `ibexa.user.invitation.group` existed with two different descriptions. Translations were regenerated

#### Checklist:
- [ ] Implement tests
- [ ] Coding standards (`$ composer fix-cs`)

### CODE REVIEW

**Review by Steveb-p — APPROVED** [review]

File: src/bundle/Resources/translations/user_invitation.en.xliff [hunk 1]
```diff
@@ -20,6 +20,21 @@
...
+        <target state="new">Invitation for '%email%' already exist.</target>
+        <note>key: user_invitation.send.invitation_exist</note>
+      </trans-unit>
+      <trans-unit id="eb29ac5a3f9317eda81b2659331df1882a03be8f" resname="user_invitation.send.success">
+        <source>Invitation send to '%email%' updated.</source>
+        <target state="new">Invitation send to '%email%' updated.</target>
+        <note>key: user_invitation.send.success</note>
+      </trans-unit>
+      <trans-unit id="7b61e31ce34b47f88e563ed2750171a834a4a589" resname="user_invitation.send.user_exist">
+        <source>User with '%email%' already exist.</source>
```
Comment (konradoboza) [inline src/bundle/Resources/translations/user_invitation.en.xliff:34]:
```suggestion
        <source>User with '%email%' already exists.</source>
```

File: src/bundle/Resources/translations/user_invitation.en.xliff [hunk 2]
```diff
@@ -20,6 +20,21 @@
         <target state="new">You are invited to join</target>
         <note>key: ibexa.user.invitation.mail.subject</note>
       </trans-unit>
+      <trans-unit id="c5663e8492fee255f3e2eb99457cf1f6d22d8967" resname="user_invitation.send.invitation_exist">
+        <source>Invitation for '%email%' already exist.</source>
+        <target state="new">Invitation for '%email%' already exist.</target>
+        <note>key: user_invitation.send.invitation_exist</note>
+      </trans-unit>
+      <trans-unit id="eb29ac5a3f9317eda81b2659331df1882a03be8f" resname="user_invitation.send.success">
+        <source>Invitation send to '%email%' updated.</source>
```
Comment (konradoboza) [inline src/bundle/Resources/translations/user_invitation.en.xliff:29]:
```suggestion
        <source>Invitation sent to '%email%' updated.</source>
```
but maybe I am missing the context here.

File: src/bundle/Resources/translations/user_invitation.en.xliff [hunk 3]
```diff
@@ -20,6 +20,21 @@
...
+        <target state="new">Invitation send to '%email%' updated.</target>
+        <note>key: user_invitation.send.success</note>
+      </trans-unit>
+      <trans-unit id="7b61e31ce34b47f88e563ed2750171a834a4a589" resname="user_invitation.send.user_exist">
+        <source>User with '%email%' already exist.</source>
+        <target state="new">User with '%email%' already exist.</target>
```
Comment (konradoboza) [inline src/bundle/Resources/translations/user_invitation.en.xliff:35]:
```suggestion
        <target state="new">User with '%email%' already exists.</target>
```

File: src/bundle/Resources/translations/user_invitation.en.xliff [hunk 4]
```diff
@@ -20,6 +20,21 @@
         <target state="new">You are invited to join</target>
         <note>key: ibexa.user.invitation.mail.subject</note>
       </trans-unit>
+      <trans-unit id="c5663e8492fee255f3e2eb99457cf1f6d22d8967" resname="user_invitation.send.invitation_exist">
+        <source>Invitation for '%email%' already exist.</source>
```
Comment (konradoboza) [inline src/bundle/Resources/translations/user_invitation.en.xliff:24]:
```suggestion
        <source>Invitation for '%email%' already exists.</source>
```

**Review by konradoboza — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**